lib/pre_assembly/digital_object.rb

```diff
@@ -89,11 +89,13 @@ def stager(source, destination)
       end
     end
 
+    # set this object's content_md_creation_style
     def content_md_creation_style
-      # set this object's content_md_creation_style
-      if (@project_style[:should_register]) || (!@project_style[:content_tag_override]) || (@project_style[:content_tag_override] && content_type_tag.blank?) # if this object needs to be registered or has no content type tag for a registered object, use the default set in the YAML file
+      # if this object needs to be registered, we will set the content type specified in YAML config
+      # if this object does NOT to be registered (:should_register == false) and the user has NOT asked for overrides (content_tag_override == false), we will also just set the content type specified in YAML config
+      if @project_style[:should_register] || (!@project_style[:should_register] && !@project_style[:content_tag_override])
```

Comment:
Could this be simplified to `@project_style[:should_register] || !@project_style[:content_tag_override]` ?